conf/messages.en

```diff
@@ -140,10 +140,10 @@ contactPhone.title.organisation = What is their telephone number?
 contactPhone.title.individual = Can we contact you by telephone?
 contactPhone.heading.organisation = What is the telephone number for {0}?
 contactPhone.heading.individual = What is your telephone number?
-contactPhone.hint = Enter a telephone number, like 01632 960 001, 07700 900 982 or +44 0808 157 0192.
+contactPhone.hint = Enter a telephone number, like&nbsp;01632&nbsp;960&nbsp;001, 07700&nbsp;900&nbsp;982 or +44&nbsp;808&nbsp;157&nbsp;0192.
```

Comment:
We want to use non-breaking spaces instead of normal spaces for phone numbers, so they the digits don't get split across lines. Could be confusing for users.